build.gradle.kts

```diff
@@ -11,6 +11,8 @@ plugins {
     id("io.spring.dependency-management") version "1.0.9.RELEASE" apply false
     kotlin("jvm") version "1.3.71" apply false
     kotlin("plugin.spring") version "1.3.71" apply false
+    id("org.jetbrains.kotlin.plugin.jpa") version "1.3.71"
```

Comment:
Why do we want to have this is in this build.gradle file as well? We only need it in coffee-backend/ ?

Reply:
I'm trying to pin the versions of libraries by defining them in the upper level `build.gradle`, so that we only need to specify the library and not the version in the lower level build files. 

Should probably add `apply false` though, så that it is not applied for all modules, only pins the version.

Reply:
Pinning the version makes it easier to update dependencies across modules in the future